ansible/server-state-playbooks/nightshade-web/templates/omero-web-config-for-webapps.j2

```diff
@@ -2,7 +2,23 @@
 # Add web-extension OMERO.web configuration
 # after web already installed and running via role
 
-config set -- omero.web.apps '["figure"]'
-config set -- omero.web.ui.top_links '[["Data", "webindex", {"title": "Browse Data via Projects, Tags etc"}], ["History", "history", {"title": "History"}], ["Help", "http://help.openmicroscopy.org/", {"target": "new", "title": "Open OMERO user guide in a new tab"}], ["Figure", "figure_index", {"target": "new", "title": "Open OMERO.Figure in a new tab"}] ]'
```

Comment:
It certainly helps when merging >1 playbook with some plain copies and some templating. A 'file' is a template which just doesn't change very much?